packages/babel-plugin-proposal-optional-chaining/test/fixtures/execute/delete.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
```

Comment:
those added tests files are using the same source code, maybe we could put this file as `fixtures/general/delete/exec.js` so if in the future any of those need an update it will be easier to locate others that need an update too

if you decide to do this, I would prefer using the very same code in `input` and `exec` (except asserts of course), that way 2 files can be easily compared when viewed alongside

Reply:
Done.